draft-ietf-quic-ack-frequency.md

```diff
@@ -119,15 +119,20 @@ endpoint performance in the following ways:
 - Similarly, receiving and processing UDP packets can also be CPU intensive, and
   reducing acknowledgement frequency reduces this cost at a data sender.
 
-- For severely asymmetric link technologies, such as DOCSIS, LTE, and satellite
-  links, connection throughput in the forward path can become constrained
-  when the reverse path is filled by acknowledgment packets. When traversing
-  such links, reducing the number of acknowledgments can achieve higher
-  connection throughput.
+- Some network technologies, such as DOCSIS, LTE, and satellite 
+entangle link transmission in the forward direction and return 
+directions. This occurs when both directions share capacity from a common link
+resource pool; significantly more resource is required
+to send in one direction that the other; or because design decisions result in
+much more capacity being available in one direction [RFC3449]. When traversing
+  this type of path, reducing the number of acknowledgments can achieve higher
+  connection throughput, lower the impact on other flows or optimise the 
+  overall use of transmission resources [REF-/doi.org/10.1002/sat.1466]. 
```

Comment:
For some network technologies, such as DOCSIS, LTE, and satellite 
link transmission in the forward direction may depend on transmission in the return 
direction. This occurs when both directions share capacity from a common link
resource pool, or when significantly more resource is required
to send in one direction that the other, or because design decisions result in
much more capacity being available in one direction {{?RFC3449}}. When traversing
  this type of path, reducing the number of acknowledgments can achieve higher
  connection throughput, lower the impact on other flows or optimise the 
  overall use of transmission resources {{SAT-QUIC}}.